### ISSUE TITLE
Can't print complete Bootstrap Modal dialog

### ISSUE THREAD

**Issue by cheborneck** [jasonday/printThis#71]
Trying to use the bootstrap modal as a print container. I set printThis in debug and the frame section is there with everything in it. The printed result however is it shows what's on the screen (the modal) and it's scrollbar, not the whole content (or frame).

**Comment by tango2310** [comment]
Does that mean that this wont work with say Magnific Lightbox?

**Comment by caseyjhol** [comment]
> A url, or html file, is neccessary to debug. Due to the complexities of printing and this plugin, an example is the best way to debug

**Comment by jasonday** [comment]
@tango2310  this plugin can print most html save for canvas objects (maps with route lines). When it comes to specific plugins, etc., it really comes down to identifying the right container to print.

**Comment by jasonday** [comment]
Closing ticket, as there is no code to debug.

**Comment by oculus42** [comment]
As an aside: If there is not a nested container to select (I'm pretty sure there is), you could likely also solve this with CSS overriding the sizes and overflow settings.

**Comment by mykhie** [comment]
Did anyone ever get a solution to this?..am experiencing the same issue

**Comment by oculus42** [comment]
@mykhie Can you provide a code example on JSFiddle or CodePen? Having a starting point would go a long way toward supporting the issue.

**Comment by mhagesfeld** [comment]
@mykhie @cheborneck I am pretty sure this is an overflow issue with the bootstrap modal.  Here is the solution I implemented:

1. Added a 'PrintContent' div with style="display:none"
2. On loading the modal content, I set the innerHTML of PrintContent to the innerHTML of the modal div
3. My print function:
`function printDealList() {
	$("#PrintContent").show();
	$("#PrintContent").printThis({
		afterPrint: function () {
			$("#PrintContent").hide();
		}
	});
	
}`

This way it does not have the limitations of the modal div.
